docs/src/js/main.js

```diff
@@ -248,6 +248,27 @@ document.querySelectorAll('.progress-bar-reset').forEach(function(el) {
     });
 });
 
+// STAR RATING SELECT
+const starRadios =  document.querySelectorAll('.star-rating-select input[name="radio-star-rating-select"]');
+
+starRadios.forEach((radio, index) => {
+    radio.addEventListener('change', function () {
+        const starSelectedClass = 'star-rating-select_filled';
+
+        // clear visual filled marks from all stars previously set
+        starRadios.forEach((_radio, _index) => {
+            starRadios[_index].classList.remove(starSelectedClass);
+        });
+
+        let i = 0;
+
+        // set visual filled marks for all stars before selected one
+        for (; i < index; i += 1) {
+            starRadios[i].classList.add(starSelectedClass);
+        }
```

Comment:
I think this could be simplified
We can do: 
```
        starRadios.forEach((current, i) => {
            current.classList.toggle(starSelectedClass, i < index);
        });
```
Also I'm not a big fan of doing _index like this, since _ is usually reserved for privates. 
I think

Reply:
Nice suggestion. Didn't know toggle had that second arg - very powerful.